docs/rackhd/authorization.rst

```diff
@@ -55,5 +33,84 @@ The following Privileges are built-in to RackHD:
       - Used to specify an ability to configure aspects of other users
     * - ConfigureSelf
       - Used to specify an ability to configure aspects of the logged in user
+    * - ConfigureManager
+      - Used to specify an ability to configure Manager resources
+    * - ConfigureComponents
+      - Used to specify an ability to configure components managed by this service
+
+Roles
+~~~~~
+
+A role grants a set of privileges.  Each privilege is specified explicitly within the role.
+Authenticated users have a single role assigned to them.
+
+
+Built-in Roles
+^^^^^^^^^^^^^^
+
+The following Roles are built-in to RackHD:
+
+.. list-table::
+    :widths: 20 100
+    :header-rows: 1
+
+    * - Role
+      - Description
+    * - Administrator
+      - Possess all built-in privileges
+    * - ReadOnly
+      - Possess Read, Login and ConfigureSelf privileges
+    * - Operator
+      - Possess Login, ConfigureComponents, and ConfigureSelf privileges
+
+API Commands for Roles
+^^^^^^^^^^^^^^^^^^^^^^
+
+The following API commands can be used to view, create, modify and delete roles.
+
+
+**Get a list of all roles currently stored in the system**
+
+::
+
+    GET /api/2.0/roles
+
+**Get information about a specified role.**
+
+::
+
+    GET /api/2.0/roles/<name>
+
+**Create a new role and store it.**
+
+::
+
+    POST /api/2.0/roles
+
+    {
+        "privileges": [
+                        <privilege1>,
+                        <privilege2>
+                      ]
+        "role": "<name>"
+    }
+
+**Modify the properties of a specified role.**
+
+::
+
+    PATCH /api/2.0/roles/<name>
+
+    {
+        "privileges": [
+                        <privilege1>,
+                        <privilege2>
+                      ]
+        "role": "<name>"
```

Comment:
It looks like, as @zyoung51 mentioned above, "name" in the patch body isn't used in the code, so I suppose it could be eliminated (@geoff correct me if I'm wrong).